content/writing_tests/screenshot.md

```diff
@@ -64,15 +64,122 @@ To reproduce this scenario, you need to capture screenshots of the egg and the p
     try {
         await _navigateTo(url);
         await env.setSimilarity(0.8);
-        await screen.find("source_egg.png").mouseMove().mouseDown(MouseButton.LEFT);
-        await new Region(0, 0, 10, 10).mouseMove();
-        await new Region(500, 700, 50, 100).mouseMove();
-        await screen.find("target_pan.png").mouseMove().mouseUp(MouseButton.LEFT);
+        await screen
+            .find("source_egg.png")
+            .mouseMove()
+            .mouseDown(MouseButton.LEFT)
+            .setX(0)
+            .setY(0)
+            .mouseMove()
+            .move(200, 700)
+            .mouseMove();
+        await screen.find("target_pan.png")
+            .mouseMove()
+            .mouseUp(MouseButton.LEFT);
         await _wait(3000);
     } catch (e) {
         await testCase.handleException(e);
     } finally {
         testCase.saveResult();
     }
-})().then(done);
+})();
+{{< /highlight >}}
+
+## How to Control the Mouse in a Sakuli Test
+
+Before using a `region` it is best to instantiate a `region` over the entire display, to use it multiple times shown in
+the examples above.
+
+### Moving the mouse to a screenshot
+
+To find a screenshot on the screen, you can use `find` or `waitForImage`. With `waitForImage` you can specify a timeout
+in which Sakuli continuously searches for the screenshot. This is useful, when e.g. starting a program like excel.
+
+{{< highlight typescript >}}
+await screen.find("search_this_screenshot.jpg")
+    .mouseMove();
+    
+//searches the screen for 5 seconds for the excel homescreen
+await screen.waitForImage("excel_homescreen.jpg", 5000)
+    .mouseMove();
+{{< /highlight >}} 
+
+`find` and `waitForImage` returns the found screenshot as a region, so chaining as follows might not work, if the second
+screenshot is not in the region of the first one.
+{{< highlight typescript >}}
+await screen.find("google_search.png")
+    .mouseMove()
+    .find("browser_address_bar.jpg")
+    .mouseMove();
+{{< /highlight >}}
+
+### Moving the mouse relative to current position
+A method to change the position is to use the `move()` method. This allows us to shift the used region, e.g.
+{{< highlight typescript >}}
+await screen.find("login_mask.png")
+    .mouseMove()
+    .click()
+    .type("MyUsername")
+    //shifts the region 50px down to e.g. a password field
+    .move(0,50)
+    //moves the mouse according to the shifted region
+    .mouseMove()
+    .click()
+    .type("MyPassword")
+    //shifts the region 50px down and 50px to the right and moves the mouse to e.g. the login button
+    .move(50,50)
+    .mouseMove()
+    .click();
+})();
+{{< /highlight >}}
+
+
+### Moving the Mouse to an Absolute Position
+
+As mentioned before, you need to create a region with a width and heigth of 1 and your x/y coordinates.
```

Comment:
Where did we mention to create a region with a width and heigth of 1 and our x/y coordinates? 😃

Reply:
I think i removed the part where it was mentioned and forgot to update this :see_no_evil: